### PR TITLE
Changes the "close" button for a "view image" link

### PR DESCRIPTION
**Pull request by javierarce**
This PR changes the "close" button for a "view image" link. Clicking on the link opens the image in a new window and closes the dialog.

Original ticket: #4937

### CODE REVIEW

**Comment by javierarce** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5709/))

**Comment by javierarce** [comment]
:eyes: can you have look, @viddo?

File: lib/assets/javascripts/cartodb/common/dialogs/static_image/export_image_result_view.js [hunk 1]
```diff
@@ -62,4 +63,7 @@ module.exports = BaseDialog.extend({
     $(e.target).focus().select();
   },
 
+  _onOpenImage: function(e) {
+    this.close();
```
Comment (viddo) [inline lib/assets/javascripts/cartodb/common/dialogs/static_image/export_image_result_view.js]: I think you could remove this method and just change the modified line above to `'click .js-open-image': 'close'` above to achieve the same, right?

**Comment by viddo** [comment]
One detail, otherwise :+1:

File: lib/assets/javascripts/cartodb/common/dialogs/static_image/export_image_result_view.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/dialogs/static_image/export_image_result_view.js]: You're totally right. I'll change it.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5716/))

**Pull request merged**